test/OpenTelemetry.Tests/Logs/LoggerProviderTests.cs

```diff
@@ -0,0 +1,116 @@
+// <copyright file="LoggerProviderTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+#if !NET461
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Exporter;
+using OpenTelemetry.Resources;
+using Xunit;
+
+namespace OpenTelemetry.Logs.Tests
+{
+    public sealed class LoggerProviderTests
+    {
+        [Fact]
+        public void VerifyDefaultBehavior()
+        {
+            var resource = CreateLoggerFactoryAndGetResource();
+
+            // Note: actual value may vary depending on test runner. Visual Studio: "unknown_service:testhost". Dotnet CLI: "unknown_service:dotnet"
+            Assert.Contains(resource.Attributes, (kvp) => kvp.Key == ResourceSemanticConventions.AttributeServiceName && kvp.Value.ToString().Contains("unknown_service"));
+        }
+
+        [Fact]
+        public void VerifyResourceBuilderAddService()
+        {
+            var resource = CreateLoggerFactoryAndGetResource(options => options.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(serviceName: "MyService", serviceVersion: "1.2.3")));
+
+            Assert.Contains(resource.Attributes, (kvp) => kvp.Key == ResourceSemanticConventions.AttributeServiceName && kvp.Value.ToString() == "MyService");
+            Assert.Contains(resource.Attributes, (kvp) => kvp.Key == ResourceSemanticConventions.AttributeServiceVersion && kvp.Value.ToString() == "1.2.3");
+        }
+
+        [Fact]
+        public void VerifyResourceBuilder_WithServiceNameEnVar()
+        {
```

Comment:
these tests are not really needed. This capability is provided by Resource, and not tied to Logging in anyway. But okay to have it as well.